libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -264,7 +264,13 @@ const AP_Param::GroupInfo AP_TECS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("PTCH_FF_K", 30, AP_TECS, _pitch_ff_k, 0.0),
 
-    // 31 previously used by TECS_LAND_PTRIM
```

Comment:
re-using slots like this is OK when they change type, and TECS_LAND_PTRIM was an AP_Int8, so this works, but we should keep a comment to say it was an AP_Int8, probably with a comment that TECS_LAND_PTRIM was removed in November 2022.

Reply:
I've update comments to make prior use clear.